src/backend/core/tests/test_models_items.py

```diff
@@ -639,6 +640,39 @@ def test_models_items__email_invitation__failed(mock_logger, _mock_send_mail):
     assert isinstance(exception, smtplib.SMTPException)
 
 
+@mock.patch("core.models.send_mail")
+@mock.patch.object(Logger, "debug")
+@override_settings(EMAIL_HOST=None)
```

Comment:
Same here, use the settings fixture.
Probably we already use the same way you made in the project, but we should use the settings fixture instead.